src/services/eventServices.js

```diff
@@ -0,0 +1,76 @@
+export function addEvent(rsoName, event) {
+    const storedEvents = getEvents(rsoName);
+    if (storedEvents) {
+        storedEvents.push(event);
+        localStorage.setItem('events:' + rsoName, JSON.stringify(storedEvents));
+    }
+    localStorage.setItem('events:' + JSON.stringify([event]));
+}
+
+export function getEvents(rsoName) {
+    const json = localStorage.getItem('events:' + rsoName);
+    if (!json) return [];
+    const parsed = JSON.parse(json);
+    parsed.forEach((v, i, arr) => {
+        arr[i] = Object.assign(new Event(), v);
+    });
+    return parsed;
+}
+
+export class Event {
+    constructor() {
+        this.name = '';
+        this.privateDescription = '';
+        this.publicDescription = '';
+        this.numParticipants = '';
+        this.repeatCycle = eventRepeatOptions.DoesNotRepeat;
+        this.date = '';
```

Comment:
Thoughts on using a JavaScript Date object here, where we'd parse the text input as a Date? We could also just validate with a regex in the event planning form, I'll leave it up to you

Reply:
I prefer the date object as well!

Reply:
Since I'm just defining which properties should be set, what do you think the default value should be? `new Date()` returns the current date/time, maybe `undefined` would be better?

Reply:
I'm fine with `undefined` being the default value